packages/nc-gui/assets/css/global.css

```diff
@@ -19,6 +19,8 @@ Apply Vazirmatn for rtl
 
 .rtl .v-application *:not(.material-icons) {
   font-family: Vazirmatn !important;
+  src: url('https://github.com/rastikerdar/vazirmatn/blob/master/fonts/ttf/Vazirmatn-Black.ttf') format('truetype');
```

Comment:
Loading font from GitHub is not a good idea

Reply:
And we are already loading the font in `assets/style/fonts.css` file

Reply:
It does not affect the interface in Windows. The font is not applied on strings.